spec/attribute.dd

```diff
@@ -617,27 +617,117 @@ auto i = 6.8;   // declare i as a double
 $(H2 $(LNAME2 scope, $(D scope) Attribute))
 
 $(P
-        The $(D scope) attribute is used for local variables and implements the
-        RAII (Resource Acquisition Is Initialization) protocol.
+        The $(D scope) attribute is used for parameters or local variables of types that contain indirections.
+        It signifies that the variable's pointer values will not escape the function
+        (in case of a $(DDSUBLINK spec/function, scope-parameters, scope parameters)) or
+        $(DDSUBLINK spec/statement, scope-statement, scope statement) that the variable is declared in,
+        for example by returning it or assigning it to a variable outside of its scope.
+)
+
+$(P
+        When applied to a global variable or a variable with a type that has no pointers, `scope` is ignored.
```

Comment:
Ok, now I see that you mention this. I would state this up-front.

Reply:
Done